queuelistener/listener.go

```diff
@@ -69,6 +71,9 @@ type Options struct {
 	// Log is used to log unexpected, non-fatal errors. It defaults to logging.DefaultLog.
 	Log logging.Logger
 
+	// Proxy protocol support
```

Comment:
Comment correctly. You should use https://github.com/golang/lint or staticcheck mentioned there for linter checks. Also I suggest reading this section: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

I just checked the project using golint and it's hell of differently commented code. So basically just stick with standard commenting practice :). `ProxyProtocol` field makes comment like `ProxyProtocol does something and insert what it does here.`
```suggestion
	// ProxyProtocol enables proxy protocol support.
```

Reply:
As I stated in the comment on initialising proxy listener - I'd suggest adding `ProxyProtocolReadHeaderTimeout` or `ProxyProtocolTimeout` or something like this that would indicate what is this timeout. Don't forget to add appropriate comment so developers will know how does it work.

Reply:
I'm also not fully sure about naming. Should it be simple `ProxyProtocol` or `EnableProxyProtocol`. I saw in the `skipper.go` `Options` struct there are fields like `EnableRouteLIFOMetrics` or `EnableRatelimiters`. So it would follow the convention there.